examples/LinkAction.main.tsx

```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from 'react';
+import { LinkAction, LinkBox, Surface } from '@itwin/itwinui-react';
+
+export default () => {
+  return (
+    <LinkBox>
+      <Surface>
+        <Surface.Header>
+          <LinkAction href='https://www.example.com'>Stadium</LinkAction>
```

Comment:
i don't like this example, it does not even look clickable.

accepting suggestions for better examples

Reply:
to clarify, i don't like that the surface does not look like something you would click on (it looks like static information). simply changing the title/description would not help.

Reply:
Maybe adding a label above asking the user to select one of the two surfaces would help. I feel the surface UI would look clickable if the title/description looks like an option to select.

Reply:
Yeah that's an option but it's too much code. I was hoping there could be a simple, self-contained example

Reply:
Let's punt on this. To increase affordance, I will use `<LinkAction as={Anchor}>` after we fix the ordering issue.

Reply:
LinkAction also does not make sense by itself. The whole point of this component is to increase the tap target size, which usually means it should be used within a container that has additional content (such as a card).